src/azure-cli/azure/cli/command_modules/network/azure_stack/commands.py

```diff
@@ -0,0 +1,1297 @@
+
+    network_nsg_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#NetworkSecurityGroupsOperations.{}',
+        client_factory=cf_network_security_groups
+    )
+
+    network_nsg_rule_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#SecurityRulesOperations.{}',
+        client_factory=cf_security_rules
+    )
+
+    network_public_ip_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#PublicIPAddressesOperations.{}',
+        client_factory=cf_public_ip_addresses
+    )
+
+    network_public_ip_prefix_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#PublicIPPrefixesOperations.{}',
+        client_factory=cf_public_ip_prefixes,
+        min_api='2018-07-01'
+    )
+
+    network_subnet_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#SubnetsOperations.{}',
+        client_factory=cf_subnets
+    )
+
+    network_vgw_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#VirtualNetworkGatewaysOperations.{}',
+        client_factory=cf_virtual_network_gateways
+    )
+
+    network_vnet_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#VirtualNetworksOperations.{}',
+        client_factory=cf_virtual_networks
+    )
+
+    network_vnet_peering_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#VirtualNetworkPeeringsOperations.{}',
+        client_factory=cf_virtual_network_peerings
+    )
+
+    network_vpn_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#VirtualNetworkGatewayConnectionsOperations.{}',
+        client_factory=cf_virtual_network_gateway_connections
+    )
+
+    network_watcher_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#NetworkWatchersOperations.{}',
+        client_factory=cf_network_watcher
+    )
+
+    network_watcher_cm_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#ConnectionMonitorsOperations.{}',
+        client_factory=cf_connection_monitor
+    )
+
+    network_watcher_flow_log_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#FlowLogsOperations.{}',
+        client_factory=cf_flow_logs,
+    )
+
+    network_watcher_flow_log_update_sdk = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.network.custom#{}',
+        client_factory=cf_flow_logs,
+    )
+
+    network_watcher_pc_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#PacketCapturesOperations.{}',
+        client_factory=cf_packet_capture
+    )
+
+    network_virtual_hub_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations#VirtualHubsOperations.{}',
+        client_factory=cf_virtual_hub,
+        min_api='2020-07-01'
+    )
+
+    network_virtual_hub_update_sdk = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.network.custom#{}',
```

Comment:
Do we need to change this?
```suggestion
        operations_tmpl='azure.cli.command_modules.network.custom#{}',
```